configuration/app-settings.md

```diff
@@ -11,9 +11,96 @@ The following settings do not need to be specified. They should only be defined
+          "offset": "4 weeks",
+          "send_day": "monday",
+          "send_time": "09:00",
+          "recipient": "reporting_unit"
+        },
+		.
+		.
+		.
+		.
+		more message definitions
+      ]
+    }
+	.
+	.
+	.
+	.
+	more schedule definitions
+]
+```
+
+|Setting|Mandatory|Description|
+|-------|---------|----------|
+|name|yes|A unique string label that is used to identify the schedule. Spaces are allowed.|
+|summary|no|Short description of the of the schedule.|
+|description|no|A narrative for the schedule.|
+|start_from|no|A date field from which exact scheduled message dates will be calculated. If not specified, the reported date will be used instead. A field from the report is usually used to determine when scheduled messages will go out.|
+|messages|yes|This is an array of settings for each message going out.<ul><li>"message": These are the actual messages set to go out in future. To achieve multiple language support, they can be set as an array of content locale pairs as in the example above, or if using version 2.15+, translation strings can be used as the message text.</li><li>"group": An integer used to categorize messages when displayed on the webapp.</li><li>"offset": A time interval from the "start_from" date that defines when the message should be sent. It can be in days or weeks.</li><li>"send_day": Day of the week the message should be sent.</li><li>"send_time": Time of day that the message should be sent in 24 hour format.</li><li>"recipient": Recipient of the message. It can be set to:<br/>'reporting_unit' - sender of the form<br/>'clinic' - clinic that the sender of the form is attached to<br/>'parent' - parent of the sender of the form.</li></ul>|
+
+
+#### 2. Set up schedule triggers
+
+Under the registrations key in app_settings, we can setup triggers for scheduled messages. A trigger for the schedule above can be defined as shown below:
+
+```
+
+"registrations": [
+{
+      "form": "PR",
+      "events": [
+        {
+          "name": "on_create",
+          "trigger": "assign_schedule",
+          "params": "ANC Visit Reminders",
+          "bool_expr": "doc.fields.weeks_lmp"
+        }
+      ],
+      "validations": {},
+      "messages": []
+    }
+]
+
+```
+
+|Setting|Mandatory|Description|
+|-------|---------|----------|
+|form|yes|Form ID that should trigger the schedule.|
+|events|yes|An array of event object definitions of what should happen when this form is received.<ul><li>'name': Name of the event that has happened. In our case, on_create happens when the form is received.</li><li>'trigger': What should happen after the named event. assign_schedule will assign the schedule named in params to this report.</li><li>'params': Any useful information for the event. In our case, it holds the name of the schedule to be triggered.</li><li>'bool_expr': A logical expression that qualifies execution of the event. In our example above, we're making sure the form has an LMP date.</li></ul>|
```

Comment:
The base text is good, but would restructure as commented for earlier table. Can you also add more info on the options for each field, and what is available in the `bool_expr`.